helm/observability-operator/values.yaml

```diff
@@ -18,6 +18,9 @@ alerting:
   grafanaAddress: ""
   slackAPIToken: ""
 
+observability:
```

Comment:
I'm confused with the observability, monitoring.opsgenieApiKey and alerting.slackApi. maybe they should all go under alerting?